### PR TITLE
feat(styles): updated vertical-align utility

### PR DESCRIPTION
**Pull request by myrta2302**

### CODE REVIEW

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

File: packages/documentation/src/stories/utilities/vertical-align/vertical-align.stories.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  render: (args: Args) => {
+    return html`<table>
+      <tbody>
+        <tr>
+          <td class="align-baseline">baseline</td>
+          <td class="align-top">top</td>
+          <td class="align-middle">middle</td>
+          <td class="align-bottom">bottom</td>
+          <td class="align-text-top">text-top</td>
+          <td class="align-text-bottom">text-bottom</td>
```
[the remark this answers is not in the document]
Comment (myrta2302) [reply packages/documentation/src/stories/utilities/vertical-align/vertical-align.stories.ts:61]: I actually borrowed these examples:  https://getbootstrap.com/docs/4.3/utilities/vertical-align/

File: packages/documentation/src/stories/utilities/vertical-align/vertical-align.snapshot.stories.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+type Story = StoryObj;
+
+export const VerticalAlign: Story = {
+  render: () => {
+    return html`
+      ${meta?.argTypes?.align?.options?.map(
+        align =>
+          html`<div>
+            <img class="logo" alt="logo" src="/assets/images/logo-swisspost.svg" />
+            <span class="align-${align}">${align ? `align-${align}` : 'text'}</span>
```
Comment (oliverschuerch) [inline packages/documentation/src/stories/utilities/vertical-align/vertical-align.snapshot.stories.ts:23]: We want to show how the image can be aligned with the text, not the other way around.

```suggestion
            <span>${align ? `align-${align}` : 'text'}</span>
            <img class="logo align-${align}" alt="logo" src="/assets/images/logo-swisspost.svg" />
```

File: packages/documentation/src/stories/utilities/vertical-align/vertical-align.styles.scss [hunk 3]
```diff
@@ -0,0 +1,59 @@
+.vertical-align-example {
+  border: 1px solid rgb(204, 204, 204);
+  line-height: 3;
```
Comment (oliverschuerch) [inline packages/documentation/src/stories/utilities/vertical-align/vertical-align.styles.scss:3]: We only want to have these styles applied on the non-table stories.

```suggestion
  :not(:has(table))& {
    border: 1px solid rgb(204, 204, 204);
    line-height: 3;
  }
```

File: packages/documentation/src/stories/utilities/vertical-align/vertical-align.styles.scss [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    padding-inline: 0.5rem;
+    &:nth-of-type(7) {
+      padding-inline-start: 2rem;
+      color: red;
+    }
+  }
+
+  .logo {
+    width: 64px;
+    height: 64px;
```
Comment (oliverschuerch) [inline packages/documentation/src/stories/utilities/vertical-align/vertical-align.styles.scss:23]:
```suggestion
    width: 1rem;
    height: 1rem;
```

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

**Review by oliverschuerch — CHANGES_REQUESTED** [review]
Good work, only one CR left.
Tell me when you solved it. Then I will approve your PR 😉

**Review by oliverschuerch — APPROVED** [review]

**Pull request merged**